packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -97,7 +99,7 @@ export const useRadio = (props: RadioProps): RadioInfo => {
       ref: buttonRef,
       ...pressable.props,
       accessibilityRole: 'radio',
-      accessibilityLabel: accessibilityLabel ?? label,
+      accessibilityLabel: (shouldAnnounceUnselected && state.selected ? '' : 'unselected') + (accessibilityLabel ?? label),
```

Comment:
This won't work outside of English languages. We don't localize strings in FURN, so either this will have to be something done by the client, or you'll need to pass this information via a prop which will localize the info on the native side.

Reply:
If the behaviour can be achieved by clients using current "accessibilityLabel" to pass "unselected" a prefix in the label, that should suffice I believe. Having a prop. like "`unselectedLabel`" and "`accessibilityLabel`" is probably avoidable in that case.

Reply:
Updating Android tester with a sample of how this can be used